README.md

```diff
@@ -32,6 +32,34 @@ Non namespaced versions are available in this release, but deprecated and will b
 
 All functions are documented in [REFERENCE.md](REFERENCE.md)
 
+### Facts
+
+All facts in this module are combined under a single toplevel `extlib` structured fact.
+
+#### `extlib['puppet_config']`
```

Comment:
If this is for the agent, suggest naming `puppet_agent_config` or `puppet_agent_settings` so it is clearly for the agent.